### PR TITLE
Health issue tooltip2

### LINKED ISSUE #1272: Add tooltips to Benefits Directory
When #1271 merges, add the tooltip as a hover to the Benefits Directory questions.

### PR DESCRIPTION
**Pull request by smcmurtry**
Closes #1272

### CODE REVIEW

File: components/tooltip.js [hunk 1]
```diff
@@ -3,52 +3,54 @@ import PropTypes from "prop-types";
...
-  :hover {
-    .${tooltipTextStyle} {
-      visibility: visible;
+class Tooltip extends Component {
+  tooltipTextStyle = css`
+    font-size: 14px;
+    font-weight: normal;
+    text-align: left;
+    visibility: hidden;
+    width: ${this.props.width}px;
```
Comment (sastels) [inline components/tooltip.js:14]: I like the way you did this 👍

File: components/tooltip.js [hunk 2]
```diff
@@ -57,9 +59,16 @@ class Tooltip extends Component {
...
+  disabled: PropTypes.bool,
+  width: PropTypes.number,
   tooltipText: PropTypes.string,
   className: PropTypes.string
 };
 
+Tooltip.defaultProps = {
+  disabled: false,
+  width: 150,
+  tooltipText: "tooltip text"
```
Comment (sastels) [inline components/tooltip.js]: Why not make tooltipText required with no default? Seems like using the default value here would always be a bug in any use of the component.

**Review by sastels — APPROVED** [review]
Minor comment, looks awesome! 🦑

File: components/tooltip.js (hunk 2 above)
Comment (smcmurtry) [reply components/tooltip.js]: good point! I'll change that.

**Pull request merged**